webservice/src/main/resources/application.properties

```diff
@@ -11,8 +11,6 @@ logging.level.org.springframework.web.filter.CommonsRequestLoggingFilter=DEBUG
 
 spring.mvc.pathmatch.matching-strategy=ant_path_matcher
 
-server.servlet.context-path=/api
```

Comment:
https://github.com/Artheanos/LN-Payments/pull/25#discussion_r810237000
![](https://media.istockphoto.com/photos/man-shooting-himself-in-the-foot-picture-id1063767128?k=20&m=1063767128&s=612x612&w=0&h=iPpdH5LKS9ZXVYsYEaeO4Xqc2WuxL8Ayaf7lHCwa1R4=)

Reply:
Tak było xd, dalej jest to fajne rozwiązanie, przez tą linijkę aby wejść do apki webowej trzebaby podać ścieżkę z api. Chyba też nie planowaliśmy wtedy bundlowania obu części tak w ogóle